compliance_checker/ioos.py

```diff
@@ -11,6 +11,7 @@
                                        get_instrument_variables,
                                        get_coordinate_variables)
 from compliance_checker import base
+from compliance_checker.cf import util as cf_util # not to be confused with cfutil.py
```

Comment:
This is confusing, but doesn't really merit a comment here -- maybe put it in the module and/or create an issue.

Reply:
I can create an issue, but the functionality is needed from the `cf/util.py` module, so at least this is unambiguously referring to that one and not `cfutil.py`. This PR isn't concerned with updating the CF modules.